.github/scripts/set-environment.sh

```diff
@@ -36,5 +36,10 @@ COMMIT_SHA=$commit_sha
 DOCKER_TAGS=$docker_tags
 APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
 
+if [[ $IS_CD ]]; then
+  echo "MIXPANEL_TOKEN=$MIXPANEL_{TIER}_TOKEN" >> $GITHUB_ENV
```

Comment:
Unless we're really picky about this I prefer the tier to not be the first thing in the name because it's more difficult to scan the list for what you're looking for.

Reply:
I agree, I generally like the variable part to be a suffix.

Reply:
But what you have here is fine too